src/app/styles/base/variables.scss

```diff
@@ -0,0 +1,7 @@
+$color-text-white: #ffffff;
+$color-background-header: #7AB72B;
+$color-box-shadow: #c7bdbd;
+$color-border: #808080;
+$color-text-item: #80808080;
```

Comment:
This value won't work on the every browser, and after AOT also.

Reply:
Please use the hex codes without alpha.

Reply:
removed